src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -20,6 +22,8 @@
  * This engine knows how to render Twig templates.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated The TwigEngine class will be removed in Symfony 4.0. You should use \Twig_Environment directly instead.
```

Comment:
`@deprecated since version 3.3, to be removed in 4.0. Use Twig instead.` (same below)